Description: Disable tests which rely on missing fixtures
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -62,38 +62,6 @@ path = "examples/pdb_symbols.rs"
 name = "stream_names"
 path = "examples/stream_names.rs"
 
-[[test]]
-name = "debug_information"
-path = "tests/debug_information.rs"
-
-[[test]]
-name = "id_information"
-path = "tests/id_information.rs"
-
-[[test]]
-name = "modi_symbol_depth"
-path = "tests/modi_symbol_depth.rs"
-
-[[test]]
-name = "omap_address_translation"
-path = "tests/omap_address_translation.rs"
-
-[[test]]
-name = "pdb_information"
-path = "tests/pdb_information.rs"
-
-[[test]]
-name = "pdb_lines"
-path = "tests/pdb_lines.rs"
-
-[[test]]
-name = "symbol_table"
-path = "tests/symbol_table.rs"
-
-[[test]]
-name = "type_information"
-path = "tests/type_information.rs"
-
 [dependencies.fallible-iterator]
 version = "0.3"
 
